mmcv/ops/box_iou_rotated.py

```diff
@@ -14,6 +18,46 @@ def box_iou_rotated(bboxes1, bboxes2, mode='iou', aligned=False):
     of bboxes1 and bboxes2, otherwise the ious between each aligned pair of
     bboxes1 and bboxes2.
 
+    coordinates details:
+        the positive direction along x axis is left->right
```

Comment:
The operator assumes: 1) the positive direction along x axis is left->right, 2) the positive direction along y axis is top->down, 3) the w border is in parallel with x axis when angle = 0. However, there are 2 xxx.